libs/designsystem/src/lib/components/empty-state/empty-state.component.ts

```diff
@@ -1,13 +1,58 @@
-import { Component, Input } from '@angular/core';
+import {
+  AfterContentInit,
+  Component,
+  ContentChildren,
+  Input,
+  OnDestroy,
+  QueryList,
+} from '@angular/core';
+import { Subject } from 'rxjs';
+import { delay, takeUntil } from 'rxjs/operators';
+
+import { ButtonComponent } from '../button/button.component';
 
 @Component({
   selector: 'kirby-empty-state',
   templateUrl: './empty-state.component.html',
   styleUrls: ['./empty-state.component.scss'],
 })
-export class EmptyStateComponent {
+export class EmptyStateComponent implements AfterContentInit, OnDestroy {
   @Input() iconName: string;
   @Input() customIconName: string;
   @Input() title: string;
   @Input() subtitle: string;
+
+  @ContentChildren(ButtonComponent)
+  private slottedButtons: QueryList<ButtonComponent>;
+
+  private ngUnsubscribe$ = new Subject<void>();
+
+  ngAfterContentInit() {
+    this.enforceAttentionLevelRules();
+
+    this.slottedButtons.changes
+      .pipe(takeUntil(this.ngUnsubscribe$), delay(0))
+      .subscribe(() => this.enforceAttentionLevelRules());
+  }
+
+  ngOnDestroy() {
+    this.ngUnsubscribe$.next();
+    this.ngUnsubscribe$.complete();
+  }
+
+  /** Enforces that all buttons will have their attention
+   * level set to 3 except the first encountered button
+   * with level 1
+   */
+  private enforceAttentionLevelRules() {
+    let encounteredLvl1Button = false;
+
+    this.slottedButtons.forEach((button) => {
+      if (button.isAttentionLevel1 && !encounteredLvl1Button) {
+        encounteredLvl1Button = true;
+      } else if (!button.isAttentionLevel3) {
+        button.attentionLevel = '3';
+      }
+    });
+  }
```

Comment:
The [issue/tech refinement](https://github.com/kirbydesign/designsystem/issues/1222#issuecomment-779841964) might not have been described well enough, but it's only supposed to be the 1st button in the list which is allowed attentionLevel1 (i.e. the wording "first of type" hinting at using `:first-of-type` pseudo selector)

Reply:
Should be resolved by 3f5f00b